src/model/RefrigerationSystem.cpp

```diff
@@ -451,6 +463,8 @@ namespace detail {
 
     if( boost::optional<RefrigerationSystem> currentSystem = refrigerationCase.system() )
     {
+      LOG(Warn, refrigerationCase.briefDescription() << " was removed from its existing RefrigerationSystem named '"
+          << currentSystem->nameString() << "'.");
```

Comment:
Add clear warn messages when enforcing unicity and removing it form any other system.